README.md

```diff
@@ -82,9 +82,13 @@ end
 ```
 
 ## Installation
+you can use it either downloading with curl,
```

Comment:
I think this would read better as "You can either install with curl ... or with antigen"